### ISSUE TITLE
Shortcodes are not working in post type

### ISSUE THREAD

**Issue by BogonFachang** [Graffino/Custom-Post-Type-Maker#24]
![image](https://github.com/Graffino/Custom-Post-Type-Maker/assets/52168802/143e5c57-f158-4e47-8d8c-d0181cfd7406)

When I add shortcodes in the post, it shows up as text instead of showing as the form.

**Comment by zenopopovici** [comment]
Hi @BogonFachang,

You have to create the shortcode in order for shortcodes to be displayed in custom post types:

https://diveinwp.com/how-to-create-shortcode-for-custom-post-type-to-display-posts/

If you're using a plugin, make sure your custom post type is selected within that particular plugin.

Regards.

**Comment by BogonFachang** [comment]
I think you have misunderstood me, Let me explain properly, I have created a custom post type called skills, inside this custom post type I created a post called skill-1 and I have added it contents to it, using classic editor. I have also formidable form, contact us form. The formidable forms have a short code. When I add that short code to the skill-1 page, it does not show the form instead shows the shortcode as it is. The shortcode works fine in pages and posts but when I add to any custom post types even new ones the shortcode fails to load as form. The same happens for custom shortcodes (to display hello world) I generated using functions.php
